test/Data/Time/PreciseDuration.purs

```diff
@@ -1,27 +1,33 @@
 module Test.Data.Time.PreciseDuration.Spec where
 
+import Data.Time.PreciseDuration
```

Comment:
We'll get a warning about multiple open imports unless we change `Prelude` (below)

Reply:
I don't get any warnings about this from either `pulp build` or `pulp test`.

Reply:
The Travis log shows them: https://travis-ci.org/awakesecurity/purescript-precise-datetime/builds/347559161#L1521-L1638

If you do `pulp build -- --censor-lib --stash`, `purescript-psa` will [persist warnings](https://github.com/natefaubion/purescript-psa#persisting-warnings).